src/components/hurricane-marker.scss

```diff
@@ -60,7 +60,10 @@
 }
 
 .category0 {
-  svg path, circle {
+  // This is based on the structure of the hurricane.svg and category-marker.svg files. Note that they're optimized
+  // by SVGR loader, so we cannot use class names. If we disable SVGR optimization, these global class names will
+  // clash with eaach other, so it's not  an option.
+  svg path:first-of-type, circle {
```

Comment:
That config looks really useful. I struggled with SVGO a few times, so if this seems like a well-tested approach, maybe we should add it to the starter-project?

Having said all that - I've just tried it here and my hurricane.svg still had its classes removed, so I couldn't change the way I'm styling its pieces.

If I completely remove SVG optimization, then there's a conflict between `.cls-1/2/3` classes defined in most of Michael's SVGs. In this particular case, my hurricane marker had a blue outline because of the style of the thermometer.svg (that is added to DOM later and `.cls-1` styles had higher priority). ;)

I'll definitely come back to this config next time I have some issues with svgr, but probably at this point not worth fighting with it in this project. Thanks for the hint though, I didn't know it's possible to have more control over this optimization process.